### ISSUE TITLE
Test controling FMI simulations with Python

### ISSUE THREAD

**Issue by jnettels** [quarree100/modeliquarree#2]
To start our work with Modelica, we need to understand what interfaces exist for the Functional Mock-Up Interface (FMI) and Python.

With 33bf438, the BouncingBall example has been expanded to show how to simulate a Functional Mock-Up Unit (FMU) with Python. For this, https://pypi.org/project/PyFMI/ can be used:

```
conda install pyfmi
```

Please note: With the first installation of PyFMI from Anaconda, the following error appeared when trying to simulate:
```
pyfmi.common.algorithm_drivers.InvalidAlgorithmOptionException: 'Invalid algorithm options object: The solver: CVode is unknown.'
```

This fixed it (updates the simulation algorithm `assimulo` to a more recent version):
```
conda install -c conda-forge assimulo
```

**Comment by jnettels** [comment]
Just a quick note to all developers in ab3 @quarree100/ab3_dev that the example for simulating FMUs with Python is now working.

**Comment by terrancelu92** [comment]
Hello,

Thanks for sharing. I updated the assimulo package as you mentioned after the first installation of PyFMI from Anaconda. However, I encountered an error as following when I am trying to simulate the fmu by running `fmu.simulate(start_time=ts, final_time=te, options=options)`:
```
File "assimulo\solvers\sundials.pyx", line 2395, in assimulo.solvers.sundials.CVode._set_atol
AssimuloException: The absolute tolerance must be positive.
```

I am currently using Python 3.7 and windows OS. I didn't install JModelica (JModelica seems to only work with Python2) but only install the PyFMI and assimulo package. 

Any hints or thoughts are greatly appreciated. 

Terrance
